### PR TITLE
Don't overwrite default opts in rich_text_area_tag

### PR DESCRIPTION
**Pull request by lmansur**
You may want to use your own controller to authenticate requests or
perform server-side validations.

### CODE REVIEW

**Review by ceritium — APPROVED** [review]

**Comment by lmansur** [comment]
Hey @zzak—sorry for the cold ping. I don't know who's maintaining _ActionText_, but I saw you participating in quite a few issues. Do you have any thoughts on this PR, or maybe point me to the right person? Thanks!

File: actiontext/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+*   Don't overwrite default opts in `rich_text_area_tag`
+
+    You may want to use your own controller to authenticate requests or
+    perform server-side validations.
```
Comment (ghiculescu) [inline actiontext/CHANGELOG.md:4]:
```suggestion
*   Allow passing in a custom `direct_upload_url` or `blob_url_template` to `rich_text_area_tag`
```

**Comment by ghiculescu** [comment]
The PR seems fine, can you please fix the merge conflict. I also made a suggestion for the CHANGELOG to make it a bit more clear.

Action Text is maintained by the core team, who maintain all features of Rails. (zzak isn't on that team, I'm not either.) They do get to these PRs but they are super busy (with Rails + real jobs) so there's no guarantee how quickly they will. Sit tight :)

**Comment by ghiculescu** [comment]
Could you squash please?

https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits

**Review by ceritium — APPROVED** [review]

**Pull request merged**